### ISSUE TITLE
Sunnarium turns into iridium ingot

### ISSUE THREAD

**Issue by frogger35** [RTRGitHub/RTRRedux#308]
Sunnarium and sunnarium parts when put in the IC2 scanner turn into a iridium ingot recipe in the memory crystal.

**Comment by Solidflip** [comment]
it makes the ingot or the recipe crystal?

**Comment by frogger35** [comment]
I'm a little confused about your question. After an item is done processing in the scanner the scanner outputs a recipe for the memory crystal based on the item put in. Well the sunnarium, sunnarium part, and recently found irradiated uranium ingot all turn into the recipe for iridium ingot in the scanner

**Comment by Solidflip** [comment]
Did some research on the issue. I replicated the issue, it appears to be anything that shares the same item id 4302. The scanner cant differentiate between the meta data apparently. I also found this: http://forum.industrial-craft.net/index.php?page=Thread&threadID=11533 Which reasserts my conclusion. 

This is "working as intended" in a way. You must use the Molecular Transformer from Adv Solars if you want to make sunnarium. Also i have no recourse in Minetweaker to correct the issue on our side.
